cluster/manager/manager.go

```diff
@@ -671,6 +674,7 @@ func (c *ClusterManager) initNodeInCluster(
 func (c *ClusterManager) joinCluster(
 	self *api.Node,
 ) error {
+	fn := "joinCluster"
```

Comment:
Ideally this should be done by the logging package itself or our own wrapper over logrus, instead of each function doing a runtime.Callers to get a `stack` instance pointer and then passing it to runtime.FuncForPC